modules/op-adding-triggers.adoc

```diff
@@ -147,6 +147,24 @@ spec:
   triggers:
     - triggerRef: vote-trigger
 ----
++
+
+Alternatively, if you have not defined a trigger custom resource, add binding and template spec to the event listener instead of referring to name of the trigger:
```

Comment:
In the previous step we tell users to copy the content and save it. To ensure continuity, suggest rephrasing to:
add binding and template spec to the EventListener YAML file, instead of referring to name of the trigger: